clients/admin-ui/src/features/datamap/datamap-table/DatamapHeaders.tsx

```diff
@@ -0,0 +1,81 @@
+import { Box, Text, Th, Thead, Tr } from "@fidesui/react";
+import { HeaderGroup } from "react-table";
+
+import { GRAY_BACKGROUND } from "~/features/datamap/constants";
+import type { DatamapRow } from "~/features/datamap/datamap.slice";
+
+const HeaderSpacer = () => (
+  <Th
+    padding={0}
+    margin={0}
+    width={4}
+    borderBottomColor="gray.200"
+    boxSizing="border-box"
+  />
+);
+
+interface DatamapHeadersProps {
+  headerGroups: HeaderGroup<DatamapRow>[];
+}
+
+const DatamapHeaders: React.FC<DatamapHeadersProps> = ({ headerGroups }) => (
+  <Thead
+    position="sticky"
+    top="0px"
+    height="36px"
+    zIndex={10}
+    backgroundColor={GRAY_BACKGROUND}
+  >
+    {headerGroups.map((headerGroup) => {
+      const { key, ...headerProps } = headerGroup.getHeaderGroupProps();
```

Comment:
this is a slight difference I had to make from datamap-ui. I think our linting rules in this repo are a little more strict. in any case, it insisted that the `<Tr>` needed a `key` which `getHeaderGroupProps` does provide, but I guess it can't infer that. so we explicitly destructure it so everyone knows we are passing it a `key` https://github.com/TanStack/table/discussions/2647